core/templates/dev/head/pages/exploration-editor-page/improvements-tab/answer-details-improvement-card/answer-details-improvement-card.directive.html

```diff
@@ -0,0 +1,9 @@
+<div>
+  <b>State Name:</b>
+  <span class="protractor-test-answer-details-state"><[getStateName()]></span>
+</div>
+<br>
```

Comment:
Do not use br tags for spacing. Use CSS classes instead.

Reply:
Done